packages/nodes-base/nodes/ConvertKit/ConvertKit.node.ts

```
@@ -0,0 +1,287 @@
+			},
+			//--------------------
+			// Field Description
+			//--------------------
+			...fieldOperations,
+			...fieldFields,
+			//--------------------
+			// FormDescription
+			//--------------------
+			...formOperations,
+			...formFields,
+			//--------------------
+			// Sequence Description
+			//--------------------
+			...sequenceOperations,
+			...sequenceFields,
+			//--------------------
+			// Tag Description
+			//--------------------
+			...tagOperations,
+			...tagFields,
+		],
+	};
+
+	async execute(this: IExecuteFunctions): Promise<INodeExecutionData[][]> {
+		const items = this.getInputData();
+		const returnData: IDataObject[] = [];
+
+		let method = '';
+		let endpoint = '';
+		const qs: IDataObject = {};
+		let responseData;
+
+		const resource = this.getNodeParameter('resource', 0) as string;
+		const operation = this.getNodeParameter('operation', 0) as string;
+
+		const fullOperation = `${resource}/${operation}`;
+
+		for (let i = 0; i < items.length; i++) {
+			//--------------------
+			// Field Operations
+			//--------------------
+			if(resource === 'field') {
+				//---------
+				// Update
+				//---------
+				if(operation === 'update') {
+					qs.label = this.getNodeParameter('label', i) as string;
+
+					const id = this.getNodeParameter('id', i) as string;
+
+					method = 'PUT';
+					endpoint = `/custom_fields/${id}`;
+				//---------
+				// Get All
+				//---------
+				} else if(operation === 'getAll') {
+					method = 'GET';
+					endpoint = '/custom_fields';
+				//---------
+				// Create
+				//---------
+				} else if(operation === 'create') {
+					qs.label = this.getNodeParameter('label', i) as string;
+
+					method = 'POST';
+					endpoint = '/custom_fields';
+				//---------
+				// Delete
+				//---------
+				} else if(operation === 'delete') {
+					const id = this.getNodeParameter('id', i) as string;
+
+					method = 'DELETE';
+					endpoint = `/custom_fields/${id}`;
+				}
```

Maybe an `else` clause to throw an error for an unknown operation?